### PR TITLE
SPS Fix

### PR DESCRIPTION
**Pull request by Kurfursten**
- tweak: Dropping an SPS no longer triggers its 'lost' text
- bugfix: Putting an SPS in a container no longer triggers its 'lost' text
- tweak: The SPS now only announces when it has been stripped (or the container it is within has been stripped) off [or, of course, you die]
- rscadd: The SPS now fits on the security belt.
- rscadd: The SPS properly reports while inside containers.
- rscadd: New proc stripped(mob,mob)

### CODE REVIEW

**Comment by Probe1** [comment]
that'll change things up.  nice one.

File: code/game/objects/items/weapons/storage/storage.dm [hunk 1]
```diff
@@ -575,4 +575,12 @@
 		else if(can_be_inserted(target))
 			handle_item_insertion(target)
 			return 1
-	return 0
\ No newline at end of file
+	return 0
+
+/obj/item/weapon/storage/OnMobDeath(mob/wearer as mob)
+	for(var/obj/item/device/gps/secure/S in contents)
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/storage.dm]: Terrible, terrible hardcode.
Also I don't think you should do this, if someone wants the SPS protection they should carry it on them, not in a bag.

File: code/game/objects/items/weapons/storage/storage.dm [hunk 2]
```diff
@@ -575,4 +575,12 @@
...
-	return 0
\ No newline at end of file
+	return 0
+
+/obj/item/weapon/storage/OnMobDeath(mob/wearer as mob)
+	for(var/obj/item/device/gps/secure/S in contents)
+		S.OnMobDeath(wearer)
+
+/obj/item/weapon/storage/stripped(mob/wearer as mob, mob/stripper as mob)
+	for(var/obj/item/device/gps/secure/S in contents)
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/storage.dm]: AUGH

File: code/modules/telesci/gps.dm [hunk 3]
```diff
@@ -15,6 +15,7 @@ var/list/SPS_list = list()
 
 /obj/item/device/gps/New()
 	..()
+	if(SPS_list.Find(src)) return //exit if we already did this
```
Comment (ComicIronic) [inline code/modules/telesci/gps.dm]: Nonono, this is an awful way to do this.
Make the next three lines into a proc, and then override that proc for the SPS.

File: code/game/objects/items/weapons/storage/storage.dm (hunk 1 above)
Comment (Kurfursten) [inline code/game/objects/items/weapons/storage/storage.dm]: It's too confusing for people that it needs to be in the pocket slot. People have been complaining for a while.

Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/storage.dm]: Then people should learn to use the item properly.
How does a health checker inside your bag even operate?

**Comment by Kurfursten** [comment]
Ready for NFD

**Pull request merged**